.github/workflows/build_deploy.yml

```diff
@@ -32,14 +32,15 @@ jobs:
 
     - name: Install dependencies
       run: |
-        pip install -r docs/requirements.txt
+        pip install -r docs/requirements.txt 
+        pip install sphinxcontrib-mermaid
```

Comment:
Since this package is included in `docs/requirements.txt`, perhaps we can remove this line?

```suggestion
```